packages/controller/lib/setup.js

```diff
@@ -2416,7 +2417,7 @@ async function processCommand(command, args, params, callback) {
                     iopckg = { version: '"' + adapter + '" not found' };
                 }
             } else {
-                iopckg = require('../package.json');
+                iopckg = require(path.join(__dirname, '../package.json'));
```

Comment:
Would prefer .. as separate arg but what’s the difference now? Btw why are we are requiring pack json and variable is named io-pack, all what is needed is version, so require js-controller-common/package json would be path independent